strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/security/ldap/LdapAuthenticatorConfigurationControllerTest.java

```diff
@@ -198,6 +200,7 @@ public void ldapConfigurationTestRequiresValidUrl()
                .body("errors[0]['messages'][0]", equalTo("must be a valid URI"));
     }
 
+    @Disabled
```

Comment:
Why was this test disabled? Is there a follow-up task for it?

cc: @steve-todorov

Reply:
I can't remember, sorry

Reply:
Try re-enabling it and checking if it's still broken. In fact, please do this for all the `@Disabled` tests and update their task descriptions.

If there isn't a task for this, could you please create one? Thanks!

Reply:
enabled and it's seems passed
thanks!

Reply:
Great! Thanks! :)

Reply:
Are there any other `@Disabled` tests that we could try, or are all the currently `@Disabled` ones not working for sure?

(Just checking).